### PR TITLE
Added 'Child nodes' section - #9

### PR DESCRIPTION
**Pull request by andrevargas**
Hello, I finished the **Child nodes** section! I hope is what you were looking for! :smile:

Feel free to request changes at any time. I reviewed it, but English isn't my native language, so, _there may be some typos_. :thinking:  Let me know if you find one.

If you liked, I can help you with other sections! :octocat:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
...
 
-> TODO
-> ViewChild vs refs
-> ContentChild vs this.props.children
+### AngularJS
+In an [AngularJS component](https://docs.angularjs.org/guide/component), we have access to the child node by injecting [`$element`](https://docs.angularjs.org/api/ng/function/angular.element) to its controller.
+```js
+class TextInputCtrl {
+  constructor($scope, $element) {
+    // $element is passed to the constructor and stored in an instance variable.
```
Comment (feimosi) [inline README.md]: No need for the comment. DI was explained before. Just add ` 'ngInject';` as in other examples.

File: README.md [hunk 2]
```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
...
+    // When the links are done, we can use the $element attribute.
+    // It has a jqLite wrapped instance of the DOM element,
+    // so we have access to some jQuery methods.
+    const input = this.$element.find('input');
+    input.on('change', console.log);
+  }
+}
+
+const component = {
+  controller: TextInputCtrl
```
Comment (feimosi) [inline README.md]: Please use `TextInputController` name.

File: README.md [hunk 3]
```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
...
+  constructor($scope, $element) {
+    // $element is passed to the constructor and stored in an instance variable.
+    this.$scope = $scope;
+    this.$element = $element;
+  }
+  
+  $postLink() {
+    // When the links are done, we can use the $element attribute.
+    // It has a jqLite wrapped instance of the DOM element,
+    // so we have access to some jQuery methods.
```
Comment (feimosi) [inline README.md]: Could you move the second sentence to the paragraph above the code snippet? Also we can mention that `$element[0]` returns the bare DOM element.

File: README.md [hunk 4]
```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
...
+  ngAfterViewInit() {
+    // The element is only avaliable when the ngAfterViewInit lifecycle hook is reached.
+    console.log(this.viewChild);
+  }
+}
+```
+
+#### ContentChild
+`ViewChild` **don't** include elements that exists within the `ng-content` tag. So we have [`ContentChild`](https://angular.io/api/core/ContentChild), that **only** includes elements that exists within the `<ng-content>` tag. [`ContentChildren`](https://angular.io/api/core/ContentChildren) does the same, but for multiple elements (also returning a `QueryList`).
+
```
Comment (feimosi) [inline README.md]: I think the description may not be clear enough for beginners. 
Take a look here:
https://stackoverflow.com/a/34327754

File: README.md [hunk 5]
```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
...
+[`ViewChild`](https://angular.io/api/core/ViewChild) returns the specified element from the view DOM. [`ViewChildren`](https://angular.io/api/core/ViewChildren) does the same, but for multiple elements (returning as a `QueryList`).
+```ts
+import {
+  Component,
+  Directive,
+  ViewChild,
+  AfterViewInit
+} from '@angular/core';
+
+@Directive({ selector: 'child' })
```
Comment (feimosi) [inline README.md]: Could be `@Component` as well, right?

**Review by feimosi — CHANGES_REQUESTED** [review]
Really good PR, thanks! Just take a look at my comments.
I'd be happy to use your help with other sections 👍

File: README.md (hunk 5 above)
Comment (andrevargas) [reply README.md]: Yeah, I just need to add the `template` or `templateUrl` metadata as well, according to the specification.

File: README.md (hunk 4 above)
Comment (andrevargas) [reply README.md]: Oh, I agree, it's really a bit confusing. I read the SO question, and I'll think in a better way to explain that for beginners and let you know when it's finished.

Comment (feimosi) [reply README.md]: Sure, let me know when you're ready. The rest looks good :+1: 
Don't forget to fix the conflicts.

Comment (feimosi) [reply README.md]: Hey, how is it going? 😄

Comment (andrevargas) [reply README.md]: Hello! Sorry for the delay. :sweat_smile:
But I think it's finished! Can you take a look? Ask me if you need more changes.

Comment (feimosi) [reply README.md]: Cool! No problem, I'll take a look 💪

**Comment by feimosi** [comment]
Looks good 👍

**Pull request merged**